news/coq-8-13-0-is-out

```diff
@@ -0,0 +1,24 @@
+<#def ID>coq-8-13-0-is-out</#def>
+<#def TITLE>Coq 8.13.0 is out</#def>
+<#def AUTHOR>Enrico Tassi</#def>
+<#def DATE>January 7th, 2021</#def>
+<#def DESCR>
+
+<p>
+The Coq development team is proud to announce the immediate availability of
+<a href="https://github.com/coq/coq/releases/tag/V8.13.0">Coq 8.13.0</a>
+</p>
+
+<p>
+Highlights:
+<ul>
+<li>Introduction of primitive persistent arrays in the core language, implemented using imperative persistent arrays. </li>
+<li>Introduction of definitional proof irrelevance for the equality type defined in the SProp sort. </li>
+<li>Many improvements to the handling of notations, including number notations, recursive notations and notations with bindings. A new algorithm chooses the most precise notation available to print an expression, which might introduce changes in printing behavior.</li>
+</ul>
+</p>
+
+
+<p>Please see <a href="https://coq.github.io/doc/v8.13/refman/changes.html#version-8-13" rel="nofollow">the changelog</a> to learn more about this release.</p>
```

Comment:
I guess you copied the HTML code from GitHub, but I would remove the `rel="nofollow"` from here :wink:

Reply:
I took it from the release of 8.12

Reply:
Ah, my bad then :wink: